examples/calling-apis/chatbot/app/(langgraph)/api/langgraph-custom-auth/[...path]/route.ts

```diff
@@ -0,0 +1,104 @@
+import { NextRequest, NextResponse } from "next/server";
+import { auth0 } from "@/lib/auth0";
+
+async function getAuthenticatedSession() {
+  const session = await auth0.getSession();
+  if (!session?.tokenSet?.accessToken) {
+    throw new Error("Unauthorized");
+  }
+  return session;
+}
+
+function addAuthContext(body: any, session: any) {
+  return {
+    ...body,
+    config: {
+      configurable: {
+        _credentials: {
+          accessToken: session.tokenSet.accessToken,
+        },
+      },
+    },
+  };
+}
+
+async function makeLangGraphRequest(
```

Comment:
I guess using the LangGraph custom auth wont work for local CLI? https://langchain-ai.github.io/langgraphjs/how-tos/auth/custom_auth/#2-update-configuration